packages/contracts/src/actions/buildContracts.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-restricted-syntax */
+
+import type { ContractsConfig } from '../types';
+
+import { buildContract } from './forc/buildContract';
+import { buildTypes } from './typegen/buildTypes';
+
+export async function buildContracts(config: ContractsConfig) {
+  for (const { path } of config.contracts) {
```

Comment:
What happens if the contracts the user wants to build are in a workspace?  If we specify the same path multiple times then it will build all of the contracts multiple times.  We need a way to build workspaces.